hushline/__init__.py

```diff
@@ -8,18 +10,37 @@
 from werkzeug.wrappers.response import Response
 
 from . import admin, routes, settings
+from .crypto import SecretsManager
 from .db import db
 from .model import User
 
 
+def _production_app_secrets_insertion(app: Flask) -> None:
+    app.config["VAULT"] = SecretsManager(
+        bytearray(getpass.getpass("admin secret: "), encoding="utf-8")
+    )
+    app.config["SECRET_KEY"] = os.environ.get("SECRET_KEY")
```

Comment:
Would you mind clarifying what you mean by this point? Do you mean line 22?

Reply:
exactly, since we narrowed the scope of the function name and `SECRET_KEY` is always derived from the environment, let's extract this line into the caller.

Reply:
Ok. I'm moving towards storing this encrypted in the database, to be retrieved during initialization. But, I think the logic will be shared for all flows, so I agree that it should be pulled out from these functions.

Reply:
Addressed by (https://github.com/scidsg/hushline/pull/411/commits/294338bb20d60ed5a2900f842e1fb432460d5d3a)